server/src/main/java/org/apache/druid/client/HttpServerInventoryView.java

```diff
@@ -187,6 +187,14 @@ public void nodeViewInitialized()
                 }
               }
 
+              @Override
+              public void nodeViewInitializedTimedOut()
+              {
+                if (!initialized.getAndSet(true)) {
+                  inventorySyncExecutor.execute(HttpServerInventoryView.this::serverInventoryInitialized);
+                }
```

Comment:
we could just call `nodeViewInitialized`

Reply:
Done